Brittany Crocker/Week 01/Calculator/calculator-part-01.js

```diff
@@ -0,0 +1,49 @@
+// # The Calculator
+//
+// ## Part 1
+// - Write a function called squareNumber that will take one argument (a number), square that number, and return the result. It should also log a string like "The result of squaring the number 3 is 9."
+const squareNumber = function (number) {
+  const result = number * number;
+  console.log(`The result of squaring the number ${number} is ${result}`);
+};
+squareNumber(10);
+
+// - Write a function called halfNumber that will take one argument (a number), divide it by 2, and return the result. It should also log a string like "Half of 5 is 2.5.".
+const halfNumber = function(num) {
+  result = num / 2;
```

Comment:
the first time you declare your result variable it is a local variable to the squareNumber function, so when you declare it here you need to use `const` or `let`.